src/cljc/athens/common_events/resolver/atomic.cljc

```diff
@@ -18,17 +18,22 @@
 
 (defmethod resolve-atomic-op-to-tx :block/new
   [db {:op/keys [args]} event-ref]
-  (let [{:block/keys [uid position]} args
-        new-block                    {:block/uid    uid
-                                      :block/string ""
-                                      :block/open   true
-                                      :block/create event-ref
-                                      :block/edits  event-ref}
-        position-tx                  (condp = (position/position-type position)
-                                       :child    (position/add-child db uid position event-ref)
-                                       :property (position/add-property db uid position event-ref))
-        tx-data                      (into [new-block] position-tx)]
-    tx-data))
+  (let [{:block/keys [uid position]} args]
+    (if (common-db/block-exists? db [:block/uid uid])
+      ;; Treast :block/new on an existing block as a :block/move instead.
```

Comment:
`Treats`

Reply:
I don't understand why we'd want to do this tho

Reply:
It's not so much "why do we want to do this" as "what do we do when this happens". Currently, when this happens, we end up with a block on two parents. That's wrong.

You can't really prevent the operation from being issued so it is more sensible to have semantics for its resolution. The two major options here are "error out" or "do something that's not wrong". I already ran into a case (repeat imports of the same content, and not just whole db imports) that I want to support, so I chose "do something that's not wrong".